mender/mender/mender.conf

```
@@ -0,0 +1,17 @@
+{
+    "HttpsClient": {},
+    "Security": {},
+    "DeviceTypeFile": "/var/lib/mender/device_type",
+    "DBus": {
+        "Enabled": true
+    },
+    "UpdatePollIntervalSeconds": 1800,
+    "InventoryPollIntervalSeconds": 28800,
+    "RetryPollIntervalSeconds": 300,
+    "TenantToken": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZW5kZXIudGVuYW50IjoiNjExNmE4YzJjYWQxZDVmMjJhODA5NDllIiwiaXNzIjoiTWVuZGVyIiwic3ViIjoiNjExNmE4YzJjYWQxZDVmMjJhODA5NDllIn0.i7Nw6LeHev04wMB_U09xYaL_C1BN177oIjgGsf4NyNclC2TNrUa-mCP-DlzeNdfoooxERzwdok1dhdphnz7o2SrPV16MxQp7CgBi-BiIeZXGd2GIrRZrb2TGjGPG5N4onvPIF-H6Ai8mxeUo5p6tJ6K5Z4-52Iuzc8KwRoj2P0hweF4mGTIz5YPj6O0zK1GSlI9rQEBTVvjYXItZV_bfAYIVoM2XSj8hw-gVNnqV8N77_uAJvEz2UqiYp_bhA1sEeDbm9j-QoC92qu8FTveRHZzjWVBG7oMZhOsh84GoF1AyL8oD4SVWlnMfG7resYSnbLWIhwA6-AB5rG7ZGpT2uRlBMltt9YWeUR0loNH0H1AnRzcU4MBi6FPV9YtC5nOXMs9Bk4ZvNkvwIyL2u86TiOWmblSNRsVBonRB7CmBneK1jnLLFnbAKcfbZXeRlh0U1_erpiF0jMnBsfySX0yZGxN6Zdmm9P38drNa6gqVfAIwy3z93af4pbYhFEnKqxSK",
```

We need to define these two partitions.
```suggestion
    "TenantToken": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZW5kZXIudGVuYW50IjoiNjExNmE4YzJjYWQxZDVmMjJhODA5NDllIiwiaXNzIjoiTWVuZGVyIiwic3ViIjoiNjExNmE4YzJjYWQxZDVmMjJhODA5NDllIn0.i7Nw6LeHev04wMB_U09xYaL_C1BN177oIjgGsf4NyNclC2TNrUa-mCP-DlzeNdfoooxERzwdok1dhdphnz7o2SrPV16MxQp7CgBi-BiIeZXGd2GIrRZrb2TGjGPG5N4onvPIF-H6Ai8mxeUo5p6tJ6K5Z4-52Iuzc8KwRoj2P0hweF4mGTIz5YPj6O0zK1GSlI9rQEBTVvjYXItZV_bfAYIVoM2XSj8hw-gVNnqV8N77_uAJvEz2UqiYp_bhA1sEeDbm9j-QoC92qu8FTveRHZzjWVBG7oMZhOsh84GoF1AyL8oD4SVWlnMfG7resYSnbLWIhwA6-AB5rG7ZGpT2uRlBMltt9YWeUR0loNH0H1AnRzcU4MBi6FPV9YtC5nOXMs9Bk4ZvNkvwIyL2u86TiOWmblSNRsVBonRB7CmBneK1jnLLFnbAKcfbZXeRlh0U1_erpiF0jMnBsfySX0yZGxN6Zdmm9P38drNa6gqVfAIwy3z93af4pbYhFEnKqxSK",
    "RootfsPartA": "/dev/mmcblk2p1",
    "RootfsPartB": "/dev/mmcblk2p2",
```